relay-event-schema/src/protocol/contexts/trace.rs

```diff
@@ -39,7 +40,19 @@ impl AsRef<str> for TraceId {
 }
 
 /// A 16-character hex string as described in the W3C trace context spec.
-#[derive(Clone, Debug, Default, Eq, Hash, PartialEq, Empty, IntoValue, ProcessValue)]
+#[derive(
+    Clone,
+    Debug,
+    Default,
+    Eq,
+    Hash,
+    PartialEq,
+    Empty,
+    IntoValue,
+    ProcessValue,
+    Serialize,
+    Deserialize,
+)]
```

Comment:
For consistency, could you please use `impl_str_serde!` instead?

https://github.com/getsentry/relay/blob/437fe72b2ece585d52652b9d0fe0311a0ff5b1d5/relay-event-schema/src/protocol/event.rs#L66

Reply:
sure thing. Done!